### ISSUE TITLE
Playlist scroll - songs swipe/delete gesture sensitivity

### ISSUE THREAD

**Issue by agmbk** [z-huang/music#1332]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Reduce the song delete gesture sensisbilty.

### Current behaviour

Open a playlist, and disable the lock option.

Swipe to scroll a playlist with an angle from vertical about 20°-30°.

Instead of scrolling a song will be swipped out, thus deleted.

### Wanted behaviour

The song delete gesture should be triggered with a swipe of at least 45°, preferably 50°-60° from vertical.

YT music solution for this functionnality could be examined as reference.

### Why do you want this feature?

The song delete gesture is overprioritized over other gestures (scroll).

A user currently have around 2 times more chance to delete a song than scrolling with a gesture (including mistake gestures, like putting the phone in a pocket etc).
This has lead to the feature request to lock a playlist, confirmation delete pop up, etc, but didn't address the core issue.

Furthermore playlist modification are made harder by this behaviour, and the lock gesture is of no help in such case.

### Additional information

- **Device:** POCO F5, Android 13, MIUI 14
- **Apk**: v.0.5.4

**Comment by DrAnonym666** [comment]
Yeah. A Playlist lock or an Option to disable the swipse gesture is the only wish i have

**Comment by nvllz** [comment]
I addressed this issue with the https://github.com/z-huang/InnerTune/pull/1175 PR, but it never made it into the app because I made it too complex. If @z-huang approves the result (there are some shots with fixed gesture sensitivity), I will try to implement it again and submit a new pull request soon.

**Comment by ruskcoder** [comment]
> I addressed this issue with the #1175 PR, but it never made it into the app because I made it too complex. If @z-huang approves the result (there are some shots with fixed gesture sensitivity), I will try to implement it again and submit a new pull request soon.

I might make PR soon, with the lock button you made, but I am trying to add a double-tap to like feature on the queue. Currently the lock button works just fine. 
![image](https://github.com/user-attachments/assets/f57cfd33-e74e-4d54-beb3-0ff1ca978675)

**Comment by MKLEGOPRO** [comment]
I use navigation gestures and I accidentally delete songs all the time, I really need this as an option that you can disable in the settings.
